.github/workflows/gajira.yml

```diff
@@ -0,0 +1,43 @@
+name: Gajira
```

Comment:
Would it be possible to name this something more descriptive? i.e. "Raise Jira Issue"